### PR TITLE
State putting

### PR DESCRIPTION
**Pull request by wlcx**
Not ready to merge, notably builds on the http stuff prq

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # Vim swap files
 *.sw?
+
```
Comment (LloydW93) [inline .gitignore:3]: Remember to ignore all line breaks.

Comment (wlcx) [inline .gitignore:3]: ಠ_ಠ

**Comment by MattWindsor91** [comment]
When is this going to be ok for merging?

**Comment by wlcx** [comment]
Was waiting for your comments and whether or not we should do all the other put stuff in here too?

**Comment by MattWindsor91** [comment]
Oh, cool, this is pull request 19.  [N-N-N-N-NINETEEN](https://www.youtube.com/watch?v=b3LdMAqUMnM)

**Pull request closed without merging**